chapters/operatorsandexpressions.tex

```diff
@@ -1402,7 +1403,13 @@ \section{Variability of Expressions}\label{variability-of-expressions}
 algorithm
   i1 := x; // Error, assignment to variable of lesser variability.
 equation
-  b2 = noEvent(x > 1); // No variability error, but equation cannot be matched.
+  /* The equation below can be rejected for two reasons:
+   * 1. Discrete-valued equation variability rule requires both sides to be
+   *    discrete-time.
+   * 2. Violates perfect matching rule, as no variable can be solved with
```

Comment:
Should it be "It violates..."? The reason 1 is given as a self-contained sentence.